pkg/cgroups/cgroups.go

```diff
@@ -45,13 +45,19 @@ func Find() (Subsystems, error) {
 	scanner := bufio.NewScanner(mountInfo)
 	for scanner.Scan() {
 		lineSegs := strings.Fields(scanner.Text())
-		if lineSegs[len(lineSegs)-3] != "cgroup" {
+		if len(lineSegs) != 11 {
```

Comment:
this seems a little less clear to me. Maybe leave a comment why we're testing 11, among the other values below